db/migrate/20221123134106_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :authorId
```

Comment:
- [ ] Kindly change to `author_id`. Ensure you run the migrations and reflect this change in the schema file.